src/components/notification/notification.ts

```diff
@@ -141,6 +141,9 @@ export class SbbNotificationElement extends LitElement {
     await this.updateComplete;
     this._setNotificationHeight();
     this._open();
+    this.shadowRoot
+      ?.querySelector<SbbNotificationElement>('.sbb-notification')!
+      .setAttribute('role', this.getAttribute('role') ?? 'status');
```

Comment:
@dauriamarco @DavideMininni-Fincons @kyubisation

Reply:
This feels a bit "hacky". Is the behavior consistent with role "status" and "alert"?
I wonder if it would be better to add accessibility documentation (e.g. add role="status" to parent)?

Reply:
I agree, I would probably remove the `role` attribute from the inner element and document which role we suggest to put on the host itself.